app/src/main/res/drawable/ic_language_icon_grey_24dp.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Don't we already have a language & settings icons on the profile chooser? I only see landscape being changed--why aren't these icons also being used in portrait?

Reply:
@BenHenning As mentioned in the previous PR with custom adapter portrait  fixes are yet to do. This PR focus only on landscape. Would be creating immediately after this PR fixes of portrait.

Reply:
Ack thanks. This might also address one of my later comments about testing portrait. I defer to you on whether you want to make sure the span count is arranged correctly for portrait in this PR or the next, but please make sure it is handled & tested.

Reply:
Okay

Reply:
> Okay will be adding them in upcoming PR of portrait mode.

Reply:
@BenHenning Okay ,
will be adding them in upcoming PR of portrait mode.